package.json

```diff
@@ -8,6 +8,7 @@
     "lint": "eslint ./src",
     "lint:fix": "eslint ./src --fix",
     "test": "react-scripts test --watchAll=false",
+    "coveralls": "react-scripts test --coverage --watchAll=false && cat ./coverage/lcov.info | coveralls",
```

Comment:
What does this do? @JiejayLan

Reply:
run the jest --coverage and then push the coverage report to my coveralls account